mapie/control_risk/ltt.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import warnings
+from numpy.typing import NDArray
+from typing import Tuple, List, Union, Optional, Any
+from .p_values import compute_hoefdding_bentkus_p_value
+
+
+def _ltt_procedure(
+    r_hat: NDArray,
+    alpha_np: NDArray,
+    delta: Optional[float],
+    n_obs: int
+) -> Tuple[List[List[Any]], NDArray]:
+    """
+    Apply the Learn-Then-Test procedure for risk control.
+    This procedure is called in multi-label-classification
+    if ``metric=precision``.
+    Note that we will do a multiple test for ``r_hat`` that are
+    less than level ``alpha_np``.
+
+    Procedure:
+        - Calculate p-values for each lambdas descretize
+        - Apply a fwer algorithm, here Bonferonni correction
+        - Return the index lambdas that give you the control
+        at alpha level
+
+    Parameters
+    ----------
+    r_hat: NDArray of shape (n_samples, )
```

Comment:
Change if necessary.
```suggestion
    r_hat: NDArray of shape (n_lambdas, )
```